doc/user/content/releases/previews.md

```diff
@@ -53,18 +53,6 @@ to physical time.
 For more information, see [Real-time
 recency](/get-started/isolation-level/#real-time-recency).
 
-### Native connectors for MySQL
-
-Materialize provides native connectors for MySQL as a source.
-
-For more information, see:
-
-- [Ingest data: MySQL](/ingest-data/mysql/)
-
-- [Create Source: MySQL](/sql/create-source/mysql/)
-
-- [Create Connection: MySQL](/sql/create-connection/#mysql)
-
```

Comment:
Moved to public preview per https://github.com/MaterializeInc/materialize/pull/29807